### ISSUE TITLE
Fantastic Pretraining Optimizers and Where to Find Them II: Hyperball Optimization

### ISSUE THREAD

**Issue by Hiroki11x** [Hiroki11x/Papers#532]
## Links

https://arxiv.org/abs/2606.16899

## Summary
この論文 “Fantastic Pretraining Optimizers and Where to Find Them II: Hyperball Optimization” は、LLM事前学習における weight decay の役割を「正則化」ではなく「重みの方向が変化する速度の制御」と捉え直し、それを明示的に制御する Hyperball という手法を提案した研究

```
一言でいうと

通常の AdamW / Muon では

weight decay → weight norm が決まる → relative update size / angular learning rate が決まる

という間接的な制御をしています。

Hyperball はこれをやめて、

weight norm を固定し、update norm も正規化することで、angular learning rate を直接制御する

という発想です。
```

[Kaiyue Wen](https://arxiv.org/search/cs?searchtype=author&query=Wen,+K), [Xingyu Dang](https://arxiv.org/search/cs?searchtype=author&query=Dang,+X), [Kaifeng Lyu](https://arxiv.org/search/cs?searchtype=author&query=Lyu,+K), [Tengyu Ma](https://arxiv.org/search/cs?searchtype=author&query=Ma,+T), [Percy Liang](https://arxiv.org/search/cs?searchtype=author&query=Liang,+P)
